lib/audit/catalog_to_moab.rb

```diff
@@ -49,6 +49,28 @@ def self.check_version_all_dirs_profiled(last_checked_b4_date)
     profiler.print_results_flat('C2M_check_version_all_dirs')
   end
 
+  def self.check_version_druid(last_checked_b4_date, druid, limit=Settings.c2m_sql_limit)
```

Comment:
Copy-pasta brain freeze.

This seems like a lot of code for this.  Maybe you just need:

```
preserved_copies = PreservedCopy.least_recent_version_audit(last_checked_b4_date).by_druid(druid)
preserved_copies.each do |pc|
    c2m = CatalogToMoab.new(pc, pc.endpoint.storage_location)
    c2m.check_catalog_version
end
```

??    Since a single druid will only have ... 50 versions x 5 endpoints  preserved copies, max?

Reply:
>.<

Reply:
Right......whoops